### PR TITLE
Update fields in source-connectors specifications: smartsheets, square, strava, tiktok-marketing, typeform, zendesk-chat, airtable, amazon-seller-partner

### LINKED ISSUE #8363: Update fields in source-connectors specifications: smartsheets, square, strava, tiktok-marketing, typeform, zendesk-chat, clickhouse-strict-encrypt, zendesk-talk, airtable, amazon-seller-partner
Add title and description properties to each field in source-connectors specifications:
source-smartsheets
source-square
source-strava
source-tiktok-marketing
source-typeform
source-zendesk-chat
source-clickhouse-strict-encrypt
**source-zendesk-talk** - _editing not needed_
**source-airtable** - _didn't appear in the source types list after the build_
**source-amazon-seller-partner** - _the title and the description are not applicable for the enum fields: AWSEnvironment  and AWSRegion_ 

[EPIC] Add title and description properties to each field in connectors specifications and order input fields appropriately #8324

### PR DESCRIPTION
**Pull request by igrankova**
## What
Add title and description properties to each field in source-connectors specifications:
source-smartsheets
source-square
source-strava
source-tiktok-marketing
source-typeform
source-zendesk-chat
source-airtable
source-amazon-seller-partner

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json [hunk 1]
```diff
@@ -9,18 +9,21 @@
     "properties": {
       "api_key": {
         "type": "string",
-        "description": "The API key for the Airtable account. https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-",
+        "description": "The API Key for the Airtable account. See the <a href=\"https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key- \">Support Guide</a> for more information on how to obtain this key.",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:12]: extra space in url ?

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json [hunk 2]
```diff
@@ -9,18 +9,21 @@
...
+		"title": "API Key",
         "airbyte_secret": true,
         "examples": ["key1234567890"]
       },
       "base_id": {
         "type": "string",
-        "description": "The base ID to integrate data from",
+        "description": "The Base ID to integrate the data from",
+		"title": "Base ID",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:20]: tabs ?

File: airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json [hunk 3]
```diff
@@ -6,57 +6,57 @@
     "type": "object",
     "properties": {
       "replication_start_date": {
-        "title": "Replication Start Date",
+        "title": "Start Date!",
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json:9]: !

**Review by grubberr — CHANGES_REQUESTED** [review]

**Comment by grubberr** [comment]
it seems for 'source_amazon_seller_partner' connector `spec.json` ignored you need to fix
```
./airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

class ConnectorConfig(BaseModel):
    class Config:
        title = "Amazon Seller Partner Spec"
```

**Review by grubberr — APPROVED** [review]

**Review by nataliekwong — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json [hunk 4]
```diff
@@ -9,18 +9,21 @@
...
+        "description": "The API Key for the Airtable account. See the <a href=\"https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-\">Support Guide</a> for more information on how to obtain this key.",
+        "title": "API Key",
         "airbyte_secret": true,
         "examples": ["key1234567890"]
       },
       "base_id": {
         "type": "string",
-        "description": "The base ID to integrate data from",
+        "description": "The Base ID to integrate the data from",
+        "title": "Base ID",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:20]: any info on where to get this base ID from?

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json [hunk 5]
```diff
@@ -9,18 +9,21 @@
...
         "examples": ["app1234567890"]
       },
       "tables": {
         "type": "array",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:24]: does the `items` property need to be set with type string?

File: airbyte-integrations/connectors/source-square/source_square/spec.json [hunk 6]
```diff
@@ -9,25 +9,29 @@
...
-        "description": "The API key for a Square application",
+        "description": "The API key for a Square application.",
+        "title": "API Key",
         "airbyte_secret": true
       },
       "is_sandbox": {
         "type": "boolean",
-        "description": "Determines the sandbox (true) or production (false) API version",
+        "description": "Determines whether to use the sandbox or production environment.",
+        "title": "Is Sandbox",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-square/source_square/spec.json:19]:
```suggestion
        "title": "Sandbox",
```

File: airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py [hunk 7]
```diff
@@ -46,10 +46,12 @@ class Config:
 
     environment: Union[ProductionEnvSpec, SandboxEnvSpec] = Field(default=ProductionEnvSpec.Config.title)
 
-    access_token: str = Field(description="Long-term Authorized Access Token.", airbyte_secret=True)
+    access_token: str = Field(description="The Long-term Authorized Access Token.", airbyte_secret=True)
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py:49]: do these have a default `title` property set?

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json (hunk 4 above)
Comment (igrankova) [reply airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:20]: A will add the description: "You can find the Base ID following the link <a href=\"https://airtable.com/api\">Airtable API</a>, log into your account, select the base you need and find Base ID in the docs."

File: airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py (hunk 7 above)
Comment (igrankova) [reply airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/spec.py:49]: Yes, on the UI all titles are set correctly.

File: airbyte-integrations/connectors/source-airtable/source_airtable/spec.json (hunk 5 above)
Comment (igrankova) [reply airbyte-integrations/connectors/source-airtable/source_airtable/spec.json:24]: Updated:
"items": {
          "type": "string"
        },

**Pull request merged**